### PR TITLE
Add tests: orb duping, 999 apples, and 999 apples + orbs

### PR DESCRIPTION
**Pull request by savage13**
Add tests:
- Orb Duping
- 999 Apples
- 999 Apples + Orb Duping

### CODE REVIEW

File: src/__tests__/dup_orbs.in.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Get 5 Diamond 1 Slate 1 Glider 4 SpiritOrb
+Save
+# Magically break 4 slots
+Break 4 Slots
+Reload
+Save
+Reload
+Sync GameData
```
Comment (iTNTPiston) [inline src/__tests__/dup_orbs.in.txt:8]: You can use the new `Init GameData` command to set gamedata to be desynced from inventory

File: src/__tests__/apples_999.out.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Initialize 1 Shield[equip] 999 Apple 1 Slate 1 Glider
+Save
+Initialize 2 Shield[equip] 999 Apple 1 Slate 1 Glider
+Break 3 Slots
```
Comment (iTNTPiston) [inline src/__tests__/apples_999.out.txt:4]: `Init GameData`

**Review by iTNTPiston — CHANGES_REQUESTED** [review]

File: src/__tests__/apples_999.out.txt (hunk 2 above)
Comment (savage13) [reply src/__tests__/apples_999.out.txt:4]: Done

File: src/__tests__/dup_orbs.in.txt (hunk 1 above)
Comment (savage13) [reply src/__tests__/dup_orbs.in.txt]: Done

**Review by iTNTPiston — APPROVED** [review]

**Pull request merged**